pkgs/top-level/release.nix

```
@@ -181,6 +181,7 @@ let
       haskell.compiler = packagePlatforms pkgs.haskell.compiler;
       haskellPackages = packagePlatforms pkgs.haskellPackages;
       idrisPackages = packagePlatforms pkgs.idrisPackages;
+      agda = packagePlatforms pkgs.agda;
```

```suggestion
      agdaPackages = packagePlatforms pkgs.agdaPackages;
```
This should fix the weird eval.

---

Want me to rebase?

---

Yes, please.